src/theme/Theme.tsx

```diff
@@ -42,6 +42,17 @@ export const Theme: Theme = {
         blue: '#B4DAF9',
         black: '#000000',
         white: '#FFFFFF',
+    },
+
+    map: {
+        street: 'mapbox://styles/mapbox/streets-v11',
+        outdoor: 'mapbox://styles/mapbox/outdoors-v11',
+        light: 'mapbox://styles/mapbox/light-v10',
+        dark: 'mapbox://styles/mapbox/dark-v10',
+        satellite: 'mapbox://styles/mapbox/satellite-v9',
+        satelliteStreet: 'mapbox://styles/mapbox/satellite-streets-v11',
+        navigationDay: 'mapbox://styles/mapbox/navigation-day-v1',
+        navigationNight: 'mapbox://styles/mapbox/navigation-night-v1',
```

Comment:
Threw in map styles into the Theme and worked pretty well with intellisense

Reply:
🤩